td-shim-tools/src/bin/td-shim-ld/main.rs

```diff
@@ -37,6 +37,12 @@ fn main() -> io::Result<()> {
                 .takes_value(true)
                 .allow_invalid_utf8(false),
         )
+        .arg(
+            arg!(-t --"payload-type" "Payload Type: [linux, executable]")
```

Comment:
please add more description on what it is.

Reply:
Done.
```
-t, --payload-type <payload-type>
          Type of payload to be launched by td-shim
          
          [default: linux]

          Possible values:
          - linux:      Payload Binary is bzImage or vmlinux, follow Linux boot protocol
          - executable: Payload Binary is a PE/COFF or ELF executable image as payload
```